glinject/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ if(ENABLE_32BIT_GLINJECT)
 	install(TARGETS ssr-glinject32 LIBRARY DESTINATION ${CMAKE_INSTALL_FULL_LIB32DIR})
 
 	set_target_properties(ssr-glinject32 PROPERTIES COMPILE_FLAGS -m32 LINK_FLAGS -m32)
+	set_target_properties(ssr-glinject32 PROPERTIES COMPILE_FLAGS -m64 LINK_FLAGS -m64)
```

Comment:
What's the purpose of this? This will break the 32-bit glinject library. If your goal is to compile only the 64-bit version, you should just set the `ENABLE_32BIT_GLINJECT` environment variable to `FALSE` before running `simple-build-and-install`.